app/views/user_mailer/send_password.html.erb

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1>Welcome to Book-a-Nook!</h1>
+    <p>
+      You have been successfully signed up to bookanook.com<br><br>
+      Your username : <b> <%= @user.email %></b><br>
+      Your password : <b> <%= @user.password %></b><br>
+    </p>
+    <p>
+      To login to the site, just follow this link: http://bookanook.com
```

Comment:
Don't hard code the link. Use routes helper, for e.g. <%= password_change_path %>